style2.css

```diff
@@ -0,0 +1,391 @@
+@media screen and (max-width: 1440px) {
+  .main__pic {
+    margin: -100px 0 0 0 ;
+	}
+}
+
+@media screen and (max-width: 1120px) {
+  .header__menu-link {
+    font-size: 14px;
+	}
+  .main__pic {
+    margin: -80px 0 0 0 ;
+	}
+}
+@media screen and (max-width: 1090px) {
+  .header__menu-link {
+	font-size: 13px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)